test/unit/spec/OAuth2Util_spec.js

```diff
@@ -1,14 +1,61 @@
 /* eslint max-len: [2, 140] */
-import Util from 'util/OAuth2Util';
+import OAuth2Util from 'util/OAuth2Util';
+import Util from 'util/Util';
+import createAuthClient from 'widget/createAuthClient';
+import Settings from '../../../src/models/Settings';
+import { AuthSdkError } from '@okta/okta-auth-js';
 
-// TODO: rewrite
+
+// TODO: add more tests
```

Comment:
what are those tests ?

Reply:
There should be tests for a happy path(i.e. [success](https://github.com/okta/okta-signin-widget/blob/master/src/util/OAuth2Util.js#L28) function is called) and a test for picking [correct](https://github.com/okta/okta-signin-widget/blob/master/src/util/OAuth2Util.js#L62) token retrieval method according to provided token options.

Reply:
Please add that as comment..

Reply:
added missing tests